dali/python/nvidia/dali/pipeline.py

```diff
@@ -804,6 +804,9 @@ def reset(self):
             self._first_iter = True
             self._last_iter = False
             self._iter = 0
+            if self._input_callbacks:
+                for group in self._input_callbacks:
+                    group.reset_indices()
```

Comment:
Isn't that a breaking change? So far we haven't reset the supplied indices.

Reply:
Maybe it is, but it seriously doubt anyone has ever used the callback with argument - I don't think it's shown in any example.